libs/components/core/src/lib/modules/scrollable-host/scrollable-host.service.ts

```diff
@@ -181,14 +220,14 @@ export class SkyScrollableHostService {
     if (element instanceof HTMLElement) {
       mutationObserver.observe(element, {
         attributes: true,
-        attributeFilter: ['class', 'style.overflow', 'style.overflow-y'],
+        attributeFilter: ['class', 'style'],
         childList: true,
         subtree: true,
       });
     } else {
       mutationObserver.observe(document.documentElement, {
         attributes: true,
-        attributeFilter: ['class', 'style.overflow', 'style.overflow-y'],
+        attributeFilter: ['class', 'style'],
```

Comment:
Curious about this change. Won't it be more performant to be more specific?

Reply:
I actually found that the more specific ones weren't working and I was just missing the test coverage to catch it. I added some tests and switched it back to just `style` as the mutation observer wasn't actually catching anything for `style.overflow` like I thought it would.